axelor-business-project/src/main/resources/views/Project.xml

```diff
@@ -53,6 +53,8 @@
         if-module="axelor-human-resource" showIf="isShowPlanning"/>
       <button name="showFinancialDatas" icon="clipboard-data" title="Show financial datas"
         onClick="save,action-project-view-show-financial-datas-panel"/>
+      <button onClick="action-project-view-attach-tasks-to-sprint"
```

Comment:
Add a save before the action in the onClick and put in next to before the sprintAllocationLinePanel directly in sprint panel + add the icon "link"